width_target: 0.5
height_target: 0.5

flow_radius: 0.03
limit_cycle_radius: 0.03

limit_cycle_speed: 10
linear_speed: 0.04
conv_rate: 10

robot_name: "ur5"
toolOffsetFromTarget: 0.7